vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -343,5 +345,31 @@ public LocalDateTime getMax() {
         return PARSER.apply(
             getElement().getProperty("max")
         );
+
+    /**
+     * Gets the internationalization object previously set for this component.
+     *
+     * Note: updating the object content that is gotten from this method will
+     * not update the lang on the component if not set back using
+     * {@link DateTimePicker#setDatePickerI18n(DatePickerI18n)}
+     *
+     * @return the i18n object. It will be <code>null</code>, If the i18n
+     *         properties weren't set.
+     */
+    public DatePickerI18n getI18n() {
+        return i18n;
+    }
+
+    /**
+     * Sets the internationalization properties for the date picker inside this component.
+     *
+     * @param i18n
+     *            the internationalized properties, not <code>null</code>
+     */
+    public void setDatePickerI18n(DatePickerI18n i18n) {
+        Objects.requireNonNull(i18n,
+                "The I18N properties object should not be null");
```

Comment:
Nit: Why "I18N" in all caps? I'd use just "i18n" in lowercase like it is in reality. Or in some contexts "I18n" if it should start with a capital, like as the first word in a sentence. Alternatively it can be unabbreviated as "internationalization".

Reply:
Updated.